src/main/rx/srxlv2.c

```diff
@@ -657,6 +691,8 @@ void srxlv2Bind(void)
         .payload = {
             .request = EnterBindMode,
             .device_id = bus_master_device_id,
+            .bind_type = DMSX_11ms,
```

Comment:
This was something that puzzled me. Should the bind type hard-coded or we should allow specifying it in CLI?
At first I thought these fields are only used in the GetBindInfo request, and filled by the receiver.

Reply:
That's a good point. I think it should be a CLI parameter that defaults to DSMX_11ms, and the "srxlv2_bind" cli command uses the value from that parameter. That way, it should still use this value by default unless the user specifies otherwise. Currently, there should be no need to use anything other than DSMX_11ms for the receivers that will currently support srxlv2.
These fields are used by both requests. For example in a multi receiver system, the FC can GetBindInfo from one receiver, and then send that info to the other receivers so they match the bind state of the initial one, without having to rebind each one.